etcdserver/api/v3rpc/key.go

```diff
@@ -134,9 +134,15 @@ func checkPutRequest(r *pb.PutRequest) error {
 	if len(r.Key) == 0 {
 		return rpctypes.ErrGRPCEmptyKey
 	}
+	if r.IgnoreLease && r.IgnoreValue {
```

Comment:
@xiang90 I thought that only revision bump will not make sense if we are not updating either the value or the lease. Since these are the two things which can be updated using both flags together doesn't make sense. What do you suggest?